anime_downloader/util.py

```diff
@@ -194,6 +194,10 @@ def format_filename(filename, episode):
 
 
 def format_command(cmd, episode, file_format, path):
+    from anime_downloader.config import Config
+    if not Config._CONFIG['dl']['aria2c_for_torrents'] and episode.url.startswith('magnet:?xt=urn:btih:'):
```

Comment:
i don't like this at all, but i can't think of an easy workaround for this right now either